client/src/components/MyOrders/TestBook.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from "react";
+import { Paper } from "@material-ui/core";
+import OrderedItem from "./OrderedItem";
+const TestBook = (props) => {
+  useEffect(() => {
+    console.log(props.checkTest.products);
+  }, []);
+  return props.checkTest.products.map((product) => {
+    if (product.course !== undefined) {
+      return (
+        <Paper
+          className="orderPageResponsive"
+          style={{
+            backgroundColor: "rgb(241, 241, 241)",
+            width: "60%",
+            display: "block",
+            margin: "100px 0!important"
+          }}
+        >
+          <OrderedItem
+            title={`${product.course.subject} Full Course`}
+            price={product.price}
+            buyDate={product.course.applicableExamDate}
+            instructor={product.course.faculty}
+          />
+        </Paper>
+      );
+    } else if (product.test !== undefined) {
+      return (
+        <Paper
+          className="orderPageResponsive"
+          style={{
+            backgroundColor: "rgb(241, 241, 241)",
+            width: "60%",
+            display: "block",
+            margin: "100px 0!important"
+          }}
+        >
+          <OrderedItem
+            title={`${product.test.subject} Full Course`}
+            price={product.price}
+            //   buyDate={product.course.applicableExamDate}
```

Comment:
Please remove the comment if not needed

Reply:
Actually this is for date field for the test series which is not yet added . Once added I would uncomment It

Reply:
Done!!